### PR TITLE
UI Dev Guide. 10 - Localization.

### PR DESCRIPTION
**Pull request by ArtsiomYeliseyenka**

### CODE REVIEW

File: app/docs/10-localization.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+At the moment Report Portal supports English (default), Russian and Belorussian languages.
+
+Localization message id format: `ComponentsName.elementName` (for example FiltersPage.addFilterButton)
+
+Localization of currently realised (Backbone) version you can find [here](https://github.com/reportportal/service-ui/tree/localization_by/src/main/resources/public/js/src/localizations).
+If you can't find some keys required for your developing component please contact [Yauheniya Labanava](https://telescope.epam.com/who/Yauheniya_Labanava).
+
+Localization workflow is:
+
+1. Develop component and define default (English) translations using `<FormattedMessage>` component or `intl.formattedMessage` function.
```
Comment (DiscoElevator) [inline app/docs/10-localization.md]: intl.formatMessage

File: app/docs/10-localization.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+For localization needs Report Portal uses [react-intl](https://github.com/yahoo/react-intl) library, which is a part of [FormatJS](https://formatjs.io/).
+
+[Documentation](https://github.com/yahoo/react-intl/wiki)
+
+All text content of application should be controlled by localization system.
+At the moment Report Portal supports English (default), Russian and Belorussian languages.
+
+Localization message id format: `ComponentsName.elementName` (for example FiltersPage.addFilterButton)
+
+Localization of currently realised (Backbone) version you can find [here](https://github.com/reportportal/service-ui/tree/localization_by/src/main/resources/public/js/src/localizations).
```
Comment (DiscoElevator) [inline app/docs/10-localization.md]: 1. realised -> implemented
2. As far as I understand this branch is out of date. Maybe it's good to mention that this branch is for BY localization only and give another link to the fresh localization folder from `develop` branch?

Comment (DiscoElevator) [reply app/docs/10-localization.md]: What do you think about adding another note telling that this branch is not consistent with the current `develop` branch and should be used for getting BY localization? For other langs it's better to look in the `develop` branch.

Comment (ArtsiomYeliseyenka) [reply app/docs/10-localization.md]: ok

File: app/docs/10-localization.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+All text content of application should be controlled by localization system.
+At the moment Report Portal supports English (default), Russian and Belorussian languages.
+
+Localization message id format: `ComponentsName.elementName` (for example FiltersPage.addFilterButton)
+
+Localization of currently implemented (Backbone) version you can find [here](https://github.com/reportportal/service-ui/tree/localization_by/src/main/resources/public/js/src/localizations).
+If you can't find some keys required for your developing component please contact [Yauheniya Labanava](https://telescope.epam.com/who/Yauheniya_Labanava).
+
+> Please use `localization_by` brunch only for getting BY localization keys. For other languages use `develop` brunch. It's caused that `localization_by` brunch is not up-to-date.
```
Comment (DiscoElevator) [inline app/docs/10-localization.md]: brunch -> branch

**Pull request merged**